lang/gcc10/Portfile

```diff
@@ -146,7 +146,9 @@ pre-configure {
         #
         # https://trac.macports.org/ticket/53726
         # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=79885
-        configure.args-append --with-sysroot="${configure.sdkroot}"
+
+        # if the sdkroot is one of the current, rapidly changing SDKS, use the generic one
+        configure.args-append --with-sysroot="[regsub {MacOSX1[1-2]\.[0-9]+\.sdk} ${configure.sdkroot} {MacOSX.sdk}]"
```

Comment:
Why limit to macOS 11 and 12. When 13 comes out in a couple of years it will break again if no one updates it.

Reply:
Yes, true, easy enough.